src/index.html

```diff
@@ -13,6 +13,91 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+
+    <div class="stars stars--0">
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+    </div>
```

Comment:
There should be empty lines between multiline sibling blocks of HTML. Add an empty line between the closing </div> of one '.stars' block and the opening <div> of the next '.stars' block.